theme/themes/eea/extras/footer.variables

```diff
@@ -2,108 +2,80 @@
-@tabletEionetLogoWidth               : 65px;
-@computerEionetLogoWidth             : 80px;
-@mobileEionetLogoMarginLeft          : 0.75rem;
-@tabletEionetLogoMarginLeft          : 0.625rem;
-@computerEionetLogoMarginLeft        : 1.563rem;
-@mobileEionetLogoMarginRight         : 0.5rem;
-@computerEionetLogoMarginRight       : 1.25rem;
-
-/* eIONET Logo Description */
-@eionetDescriptionFontWeight         : @bold;
-@mobileEionetDescriptionPaddingTop   : 0.125rem;
-@tabletEionetDescriptionPaddingTop   : 0.313rem;
-@mobileEionetDescriptionFontSize     : 0.5rem;
-@tabetEionetDescriptionFontSize      : 1rem;
-@computerEionetDescriptionFontSize   : 1.063rem;
-@mobileEionetDescriptionLineHeight   : 0.625rem;
-@tabletEionetDescriptionLineHeight   : 1.5rem;
-@computerEionetDescriptionLineHeight : 1.8rem;
+@tabletEionetLogoWidth               : 225px;
+@computerEionetLogoWidth             : 260px;
+@tabletEionetLogoMarginLeft          : @space-3;
 
 /* Contact Info */
-@mobileContactMargin                : 0.3rem 0;
-@tabletContactMargin                : 0.6rem 0;
-@computerContactMargin              : 1rem 0;
-@mobileContactFontSize              : 0.563rem;
-@tabletContactFontSize              : 0.875rem;
-@computerContactFontSize            : 1rem;
-@tabletContactIconMarginRight       : 0.938rem;
-@tabletContactIconFontSize          : 1.375rem;
-@computerContactIconFontSize        : 1.563rem;
+@mobileContactBlockMarginInline     : 10%;
+@tabletContactBlockMarginInline     : 0;
+@tabletContactBlockMarginbottom     : @space-4;
+@mobileContactMargin                : @space-4 0;
+@computerContactMargin              : @space-5 0;
+@mobileContactFontSize              : @font-size-1;
+@computerContactFontSize            : @font-size-2;
+@mobileContactIconMarginRight       : @space-3;
+@mobileContactIconFontSize          : @font-size-4;
+@computerContactIconFontSize        : @font-size-5;
 
 /* Address */
-@mobileAdressFontSize               : 0.5rem;
-@tabletAddressFontSize              : 0.875rem;
+@addressFontSize                    : @font-size-0;
 
 /* Social Icons */
 @socialIconColor                    : @white;
-@mobileSocialIconSize               : 0.9rem;
-@tabletSocialIconSize               : 1.8rem;
-@computerSocialIconSize             : 2rem;
+@mobileSocialJustifyContent         : space-around;
+@tabletSocialJustifyContent         : start;
+@tabletSocialGap                    : @space-1;
+@mobileSocialIconSize               : @font-size-5;
+@tabletSocialIconSize               : @font-size-4;
+@computerSocialIconSize             : @font-size-5;
 
 /* Footer Menu */
-@mobileMenuMargin                   : 1rem auto 0;
-@tabletMenuMargin                   : 2rem auto 0;
-@tabletMenuWidth                    : 500px;
+@mobileMenuMargin                   : @space-8 auto 0;
+@tabletMenuMargin                   : @space-13 auto 0;
+@computerMenuMargin                 : @space-20 auto 0;
+@mobileMenuWidth                    : 100%;
```

Comment:
@g-stamatis for Antonis :) I would have liked to have this section simpler with a space-evenly without the width differences but this works like it is as well.